standards/code-versioning.md

```diff
@@ -2,83 +2,110 @@
 
 ## What is Git Tagging?
 
-  * [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) is a simple, easy way to keep track of your code at a certain point in time. Some instances where this becomes useful are:
-    * Client-facing deliverables such as milestones, UAT, or production launch/release
-    * Internal-facing code at a point that you or your team wants to demarcate as important
-    * Versioning of your code (as a default, Shift3 follows SemVer)
-
-## SemVer Basics:
-  * Always have an up-to-date documented API (in our case, README.md for our projects or Wiki)
-  * Increment your versions based on a **Major.Minor.Patch** format where:
-    * **Major** changes are breaking, non-backwards compatible changes to your code base.
-    * **Minor** changes are new features or functionality that ius still backwards-compatable
-    * **Patch** changes are bug fixes or hotfixes that only address issues and don't introduce anything new.
-  * Releases should be unalterable and locked -- forever.
-  * Initial development (so most of our lives) should be treated as 0.x.x so as to indicate that there is no release as yet.
-    * Generally, any software with a major version of 0 should be treated as unstable.
+-   [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) is a simple, easy way to keep track of your code at a certain point in time. Some instances where this becomes useful are:
+    -   Client-facing deliverables such as milestones, UAT, or production launch/release
+    -   Internal-facing code at a point that you or your team wants to demarcate as important
+    -   Versioning of your code (as a default, Shift3 follows SemVer)
+
+## [SemVer](https://semver.org/) Basics:
+
+-   Always have an up-to-date documented API (in our case, README.md for our projects or Wiki)
+-   Increment your versions based on a **Major.Minor.Patch** format where:
+
+    -   **Major** changes are breaking, non-backwards compatible changes to your code base.
+    -   **Minor** changes are new features or functionality that ius still backwards-compatable
+    -   **Patch** changes are bug fixes or hotfixes that only address issues and don't introduce anything new.
+
+    ##### Example:
+
+    ![example version](./assets/qa-process/semver.png)
+    In this instance:
+
+    -   We are on the `8`th major version, so there've been 7 changes that have broken backward compatibility in the past
+    -   We are on the `2`nd minor version which means there have been two feature changes that have been made
+    -   After we did our second feature push, we have had `6` bugs that we have fixed in the meantime.
+    -   If we build a new feature, this number will be changed to `8.3.0`.
```

Comment:
I would suggest being explicit on this feature example.

```suggestion
    -   If we build a new feature that does not break existing APIs, this number will be changed to `8.3.0`.
```

Reply:
That's a great suggestion.